app/components/views/Home/index.tsx

```diff
@@ -302,6 +308,7 @@ export const Home: FC = () => {
                 onHide={() => setShowMobileMenu(false)}
               />
             </div>
+            <ThemeToggle />
```

Comment:
Let's override the base surface color to have the button color contrast with the background :)

<img width="247" alt="image" src="https://user-images.githubusercontent.com/97446324/167521197-81e5ec0a-380a-43ba-8d10-5472d4a318ad.png">


https://github.com/KlimaDAO/klimadao/blob/8a593c81489705b6b6625cefe5805272b78ba21d/site/components/pages/Pledge/PledgeLayout/styles.ts#L30-L32